src/ipa/rpi/controller/meson.build

```diff
@@ -25,6 +25,7 @@ rpi_ipa_controller_sources = files([
     'rpi/sharpen.cpp',
     'rpi/sync.cpp',
     'rpi/tonemap.cpp',
+    'rpi/clock_recovery.cpp',
```

Comment:
I think libcamera like to keep these filenames in alphabetical order, so it will probably need to go further up.